RedHatInsights/insights-behavioral-spec
# Update aggregator tests configuration variables

# Description

- Add `storage type`environment variable introduced in https://github.com/RedHatInsights/insights-results-aggregator/pull/1755 to the config files
- Export `config/insights_results_aggregator_docker_env.toml` to setup default env vars in docker environment
- Fix API prefix used in REST API scenarios
- Fix some scenario naming so output and error logs can be created properly

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- Configuration update

## Testing steps

Run `make aggregator-tests` and the database migration and REST API scenarios should pass

## Checklist
